scripts/uitwerkingen.js

```diff
@@ -14,10 +15,12 @@ console.log("Sales is een uitdagende afdeling om te werken als Verkoopmanager."
 //const userInput = prompt('Hoi! Hoe heet je?');
 //console.log(userInput);
 
+
 //opdracht 2a:
 const userInput = prompt("Over welke afdeling wil je meer informatie? Kies uit: [marketing / sales / customer-service]")
 console.log(userInput);
 
+
```

Comment:
Als ik de betekenis van @@ int @@ switch kunnen vinden had ik iets kunne leren hiervan, als dit gewerkt heeft bij jou dan is het knap dat je de alternatieven toepast maar ik kent het niet en kan hier geen oordeel over geven. In de uitwerkingen zijn de cases voluit geschreven. 
Default message is niet compleet.